src/lib/webworker.generated.d.ts

```diff
@@ -430,12 +430,10 @@ interface ImageData {
     data: number[];
     height: number;
     width: number;
+    new(width: number, height: number): ImageData;

```

Comment:
The [MDN spec](https://developer.mozilla.org/en-US/docs/Web/API/ImageData/ImageData) has only two constructors with arguments. And I tried in IE, Edge, Firefox and Chrome, none of them supports an argument-less constructor for ``ImageData``